web/docs/Frontend Starters/Next.js/Next.js + Drupal/setting-environment-variables.md

```diff
@@ -56,3 +57,48 @@ CLIENT_SECRET
 
 See [Implementing Preview](./implementing-preview.md) for more information on
 how to implement Decoupled Preview with Next.js and Drupal.
+
+## Connecting to Multidev Cloud Environments
+
+To connect to a Multidev environment, the following helper environment variable
+must be used inside of `next.config.js`.
+
+- `PANTHEON_ENVIRONMENT` - Automatically configured the value of the beginning
+  prefix (live, pr-number (pr-12), branch-name (multi-integration)) of
+  `PANTHEON_ENVIRONMENT_URL`. This is to act as a variable to key off of and
+  source data from a specific BE environment
+- `IS_LIVE_ENVIRONMENT` - Automatically set to true if
+  `PANTHEON_ENVIRONMENT_URL` is live and false otherwise
+
+Either the `PANTHEON_CMS_ENDPOINT` or `WPGRAPHQL_URL` will need to be set.
+
+Taking a look at how the `next.config.js` works, there is this logic which sets
+the `backendUrl`
+
+```
+let backendUrl, imageDomain;
+if (process.env.WPGRAPHQL_URL === undefined) {
+	backendUrl = `https://${process.env.PANTHEON_CMS_ENDPOINT}/wp/graphql`;
+	imageDomain = process.env.IMAGE_DOMAIN || process.env.PANTHEON_CMS_ENDPOINT;
+
+	// populate WPGRAPHQL_URL as a fallback and for build scripts
+	process.env.WPGRAPHQL_URL = `https://${process.env.PANTHEON_CMS_ENDPOINT}/wp/graphql`;
+} else {
+	backendUrl = process.env.WPGRAPHQL_URL;
+	imageDomain =
+		process.env.IMAGE_DOMAIN ||
+		process.env.WPGRAPHQL_URL.replace(/\/wp\/graphql$/, '').replace(
+			/^https?:\/\//,
+			'',
+		);
+}
+```
+
+To use `PANTHEON_ENVIRONMENT` and connect to a Multidev backend, this line could
+be added under the above logic and after `PANTHEON_ENVIRONMENT_PREFIX` has been
+defined
+
+```
+backendUrl = `https://${PANTHEON_ENVIRONMENT}-${process.env.WPGRAPHQL_URL.replace(/^https?:\/\//,'',)}`
```

Comment:
Does PANTHEON_ENVIRONMENT need the process.env prefix?

Reply:
That is correct and the update is staged in the current set of revisions 👍